### ISSUE TITLE
Give attribution

### ISSUE THREAD

**Issue by jacobgil** [hiveml/tensorflow-grad-cam#1]
Hi,
Nice repository and blog post about it.

It looks like its based in some form on my implementations of grad-cam.
Specifically, the code starting here:
https://github.com/hiveml/tensorflow-grad-cam/blob/master/main.py#L77
Is exact to the code in two grad-cam implementations I made:
https://github.com/jacobgil/keras-grad-cam/blob/master/grad-cam.py#L105
https://github.com/jacobgil/pytorch-grad-cam/blob/master/grad-cam.py#L111

I'm glad you found my implementation helpful,
Could you please give it proper attribution in the Readme and the blog post?

Thanks,
Jacob

**Comment by ryanjay0** [comment]
I actually derived this code from another repo:https://github.com/Ankush96/grad-cam.tensorflow which specifically implements it in raw tensorflow (not keras or pytorch). I attributed that repo in the blog and just added it to the bottom of our repo. It does seem like your repo pre-dates that one but since it's what I actually used I think this is fair. I've honestly never seen your repo till now, but i'm all about proper attribution. They refer to this repo https://github.com/ramprs/grad-cam which seems to predate yours. Thanks for the comment.
